protos/sources/vega/vega.proto

```diff
@@ -1154,6 +1166,12 @@ message MarginLevels {
   string asset = 7;
   // Timestamp in Unix nanoseconds for when the ledger entry was created.
   int64 timestamp = 8;
+  // Margin required to cover orders in isolated margin mode.
+  string order_margin = 9;
+  // Margin mode for the party, cross margin or isolated margin.
+  MarginMode margin_mode = 10;
+  // Margin factor, relevant only for isolated margin, empty otherwise.
+  string margin_factor = 11;
 }
 
 // Represents market data specific to a perpetual market.
```

Comment:
Not part of this PR, but I've spotted it so I'll comment:
the new account type `ACCOUNT_TYPE_ORDER_MARGIN` needs to be added to the big list in the API that exports ledger entries `func (ls *Ledger) Export()`.